src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -69,7 +69,11 @@ public function provideDefaultOgPermissions(PermissionEventInterface $event) {
    *   The default role event.
    */
   public function provideDefaultRoles(DefaultRoleEventInterface $event) {
-    $event->addRole(OgRoleInterface::ADMINISTRATOR, ['label' => 'Administrator']);
+    $event->addRole([
```

Comment:
How about something like

```php
public function provideDefaultRoles(DefaultRoleEventInterface $event) {
  $role = $this->ogRoleStorage->create(['name' => OgRoleInterface::ADMINISTRATOR);
  // Letting OgRole do it's job ...
  $role->setLabel('Administrator')
    ->setAdmin();

  $event->addRole($role);
}
```

Now our event's addRole doesn't need to know much about the role -- it's just passing it on. It's up to OgRole::save() to validate nothing is wrong.